### PR TITLE
Tradução para PT plugins e rest

### PR DESCRIPTION
**Pull request by alexsmonte**
Tradução para PT plugins e rest

### CODE REVIEW

**Comment by alexsmonte** [comment]
I did not understand the mistake

Warning, treated as error:
/data/pt/plugins.rst:21: WARNING: Inline strong start-string without end-string.
Makefile:42: recipe for target 'html' failed
make[1]: *** [html] Error 1
make[1]: Leaving directory '/data/pt'
Makefile:44: recipe for target 'html-pt' failed
make: *** [html-pt] Error 2

File: pt/plugins.rst [hunk 1]
```diff
@@ -1,12 +1,645 @@
...
+deve fazer assim o assim::
+
+    php composer.phar require cakephp/debug_kit
+
+Isso instalaria a versão mais recente do DebugKit e atualizaria seus arquivos **composer.json**, **composer.lock**, atualização **vendor/cakephp-plugins.php** e atualize seu autoloader.
+
+Se o plugin que deseja instalar não estiver disponível em
+Packagist.org, você pode clonar ou copiar o código do plugin para seu diretório**plugins**. 
+Supondo que você deseja instalar um plugin chamado 'ContactManager', você
+Deve ter uma pasta em**plugins**chamado 'ContactManager'. Neste diretório
```
Comment (markstory) [inline pt/plugins.rst]: You might need a space on either side of the bolded words here.

File: pt/plugins.rst [hunk 2]
```diff
@@ -1,12 +1,645 @@
...
+Ou você pode carregar os plugins individualmente::
+
+    // Carregando apenas o blog e inclui rotas
+    Plugin::load('Blog', ['routes' => true]);
+
+	// Inclua o arquivo configuration/initializer do bootstrap.
+    Plugin::load('ContactManager', ['bootstrap' => true]);
+
+Com qualquer uma das abordagens, você não precisa mais manualmente ``include()`` ou
+``Require()`` configuração de um plugin ou arquivo de rotas - acontece
```
Comment (markstory) [inline pt/plugins.rst]: Should be `require()` here.

File: pt/plugins.rst [hunk 3]
```diff
@@ -1,12 +1,645 @@
...
+        function ($routes) {
+            $routes->get('/contacts', ['controller' => 'Contacts']);
+            $routes->get('/contacts/:id', ['controller' => 'Contacts', 'action' => 'view']);
+            $routes->put('/contacts/:id', ['controller' => 'Contacts', 'action' => 'update']);
+        }
+    );
+
+
+O código acima irá conectar as rotas padrão para o seu plugin. Você pode personalizar isso
+no arquivo com rotas mais específicas mais tarde.
```
Comment (markstory) [inline pt/plugins.rst]: Missing the `::` here.

File: pt/plugins.rst [hunk 4]
```diff
@@ -1,12 +1,645 @@
...
+    {
+        public function initialize(array $config)
+        {
+            $this->hasMany('AltName', [
+                'className' => 'ContactManager.AltName',
+            ]);
+        }
+    }
+
+Você pode usar `TableRegistry`` para carregar suas tabelas de plugins usando o familiar
```
Comment (markstory) [inline pt/plugins.rst]: TableRegistry needs two backticks on the left side.

**Comment by markstory** [comment]
Looks like you are missing a link reference somewhere 

```
/data/pt/development/rest.rst:18: WARNING: undefined label: resource-routes (if the link has no caption the label must precede a section header)
```

**Comment by markstory** [comment]
Looks like there are still a few build errors. Good work so far though!

**Pull request merged**